packages/arb-token-bridge-ui/src/hooks/__tests__/useUpdateUsdcBalances.test.ts

```diff
@@ -0,0 +1,258 @@
+import { renderHook } from '@testing-library/react'
+import { registerCustomArbitrumNetwork } from '@arbitrum/sdk'
+
+import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
+import { ChainId } from '../../util/networks'
+import { CommonAddress } from '../../util/CommonAddressUtils'
+
+import { getProviderForChainId } from '@/token-bridge-sdk/utils'
+import {
+  childChainUsdcAddressFetcher,
+  useParentChainUsdcAddress
+} from '../CCTP/useUpdateUsdcBalances'
+import { useNetworks } from '../useNetworks'
+import { orbitMainnets, orbitTestnets } from '../../util/orbitChainsList'
+import { getL2ERC20Address } from '../../util/TokenUtils'
+
+jest.mock('../useNetworks', () => ({
+  useNetworks: jest.fn()
+}))
+
+jest.mock('../../util/TokenUtils', () => ({
+  getL2ERC20Address: jest.fn()
+}))
+
+const xaiTestnetChainId = 37714555429
```

Comment:
nit: you could cast all of these `as ChainId` here and won't have to do it every time later